cmd/root.go

```diff
@@ -114,6 +113,7 @@ func validateTopology(cmd *cobra.Command, args []string) error {
 
 var (
 	topoNew = topo.New
```

Comment:
I don't think this stubbing is needed anymore, its not used in tests either. So probably best to streamline the params to not need a load/new func